### PR TITLE
Use case edit

### PR DESCRIPTION
**Pull request by woodbe**
Edit of the use case to point out that CMFA is also used for maintaining the unlocked state of the device.

### CODE REVIEW

**Comment by woodbe** [comment]
Based on the last call, should the use case state that CMFA is used for authentication, meaning the "initial" authentication to the device, or should it be that it maintains the authentication state (and hence the lock/unlock state) for the device? 

This also impacts the definition used for CMFA

File: CMFA_ESR.adoc [hunk 1]
```diff
@@ -98,10 +98,10 @@ When used in a security system, the biometric product needs to take into account
 == Use Case(s)
 CMFA products are used primarily for user authentication for mobile devices such as smartphones, where the authentication can be used to unlock the mobile device. This status can also be provided to external services such as a PC login at the office, building or room entrance control or ATMs.
 
-The first version of the PP-Module focuses on the use case that the CMFA product is used for unlocking the mobile device. Additional PP-Modules have to be created for other use cases. 
+The first version of the PP-Module focuses on the use case that the CMFA product is used for unlocking (and maintaining the appropriately unlocked state of) the mobile device. Additional PP-Modules have to be created for other use cases.
```
Comment (gfiumara) [inline CMFA_ESR.adoc]:
```suggestion
The first version of the PP-Module focuses on the use case that the CMFA product is used for unlocking and maintaining the appropriately unlocked state of the mobile device. Additional PP-Modules have to be created for other use cases. 
```

**Review by gfiumara — APPROVED** [review]
Agree that it should be both the _initial_ auth + maintaining the auth. Removing the parenthesis would make that explicit.

**Review by gregott — APPROVED** [review]

**Comment by woodbe** [comment]
Mark this to be reviewed in #24 by @woodbe

**Comment by woodbe** [comment]
Closing as it looks like this is covered in the update

**Pull request closed without merging**